admin/controllers/retro-input-assistant.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const checkWindowV2Service = require('../services/check-window-v2.service')
+const businessAvailabilityService = require('../services/business-availability.service')
+const ValidationError = require('../lib/validation-error')
+const retroInputAssistantService = require('../services/retro-input-assistant.service')
+
+const controller = {}
+
+controller.getAddRetroInputAssistant = async (req, res, next, error = null) => {
```

Comment:
It would be better practise, in my view, if we used named functions, as at least then, should we choose to use an APM, the functions will be meaningful.